blocks/eda-frontend/src/components/SchematicEditor/ComponentProperties.js

```diff
@@ -112,6 +112,61 @@ function adjustPorts (newPorts, offsetPorts, newTotalPorts, oldPorts, block, por
   }
 }
 
+const getHelperText = (value) => {
+  switch (value) {
+    case 1:
+      return 'boolean is expected'
+    case 2:
+      return 'integer is expected'
+    case 3:
+      return 'double is expected'
+    case 4:
+      return 'double with scale is expected'
+    case 5:
+      return 'complex is expected'
+    case 6:
+      return 'string is expected'
+    case 7:
+      return 'yesno is expected'
+    case 8:
+      return 'filename is expected'
+    case 9:
+      return 'vector of booleans is expected'
+    case 10:
+      return 'vector of integers is expected'
+    case 11:
+      return 'vector of doubles is expected'
+    case 12:
+      return 'vector of doubles with scale is expected'
+    case 13:
+      return 'vector of complexes is expected'
+    case 14:
+      return 'vector of strings is expected'
+    case 15:
+      return 'vector of yesnoes is expected'
+    case 16:
+      return 'vector of filenames is expected'
+    case 17:
+      return 'array of booleans is expected'
+    case 18:
+      return 'array of integers is expected'
+    case 19:
+      return 'array of doubles is expected'
+    case 20:
+      return 'array of doubles with scale is expected'
+    case 21:
+      return 'array of complexes is expected'
+    case 22:
+      return 'array of strings is expected'
+    case 23:
+      return 'array of yesnoes is expected'
+    case 24:
+      return 'array of filenames is expected'
+    default:
+      return ''
+  }
+}
```

Comment:
```suggestion
const helperText = {
  1: 'boolean is expected',
  2: 'integer is expected',
  3: 'double is expected',
    ...
}

const getHelperText = (compType) => {
  if (compTye in helperText) {
    return helperText[compType];
  }
  return ''
}
```
first make a helperText object. use that in getHelperText.